model/dbCommands.js

```diff
@@ -20,6 +20,15 @@ function wrapDB (dbconfig) {
 
 const db = wrapDB(dbconfig);
 
+exports.getJobRoles = async () => { 
+    try{
+        return await db.query( 
+            "SELECT Role_ID, JobRoles.Name, Level FROM JobRoles INNER JOIN Band ON JobRoles.Band_ID = Band.Band_ID ORDER BY Level");
+    } catch(e) {
+        throw new DatabaseError(`Error calling getCapabilitiesBasedOnJobId with message: ${e.message}`);
```

Comment:
Change `Error calling getCapabilitiesBasedOnJobId with message: ${e.message}` to `Error calling getJobRoles with message: ${e.message}`